### DISCUSSION TITLE
Authentication code not received via email

### DISCUSSION THREAD

**Discussion by vlijmenfileer** [community/community#118929] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hi,

After quite a while of not using it, I wanted to relogin to one (of two) GitHub accounts I have.  I am greeted with a text stating I will be sent an email with a code.  The email however does not arrive:
"We just sent your authentication code via email to <removed>. The code will expire at 3:16PM CEST."

My email account is correct, it is working, I tried multiple times over the past hours.
Does anybody have an idea what could be happening?

I'd have to add that I found a string of emails from last December and Januray urging me to enroll 2FA.  These also stated "On January 19th, 2024 at 00:00 (UTC) your account will be required to have 2FA for authentication. If you have not yet enrolled by that date, your ability to access GitHub.com will be limited until you finish the enrollment process."  So even though that's a while back there is the guarantee I will be able to get back in, and presumably be forced to enroll 2FA.

**Comment by pulsationseutrr** [comment]
Same. Since this morning I can't connect to my second Github because I don't receive the authentication code.

**Reply by thibaut-lo** [reply]
Same for me.

**Reply by aminulloh2002** [reply]
me too

**Reply by im-divxnshh** [reply]
same prob

**Reply by Lukyrouge3** [reply]
I received it 5h after asking for it...

**Comment by rickscsharp** [comment]
Same for me.

**Comment by vlijmenfileer** [comment]
Pfff.  Now I have multiple emails in my Inbox, must have been a short outage.  Or maybe there's just a few hours delay in this service ...

**Comment by deno-snek** [comment]
Is the problem solved?
This is frustrating. I have the same problem. Heck I wanted to comment on this thread and I realized I can't login in the first place.

**Comment by Strawberrydog32** [comment]
Same issue here.

**Comment by butter-fly04** [comment]
Same here.

**Comment by grebba** [comment]
Same for me

**Comment by bzeiss** [comment]
Same problem for me since Saturday.

**Comment by hakrichTech** [comment]
the same issue for me too i tried almost for an hour now

**Comment by Lzar17** [comment]
same issue here. How to solve😭

**Reply by hakrichTech** [reply]
there is no way of solving this issue until our fellow developers from github come back from weekend party. they are the one the key for this issue

**Reply by Lzar17** [reply]
About 4 hours ago, I try to sign in. But I received many e-mails about verification two minutes ago…too late

**Comment by sanity-metal** [comment]
same for me, i crated a new account with a microsoft email just to be able to log in...

**Comment by donerre2** [comment]
same with my acc @donerre since yesterday morning. i had to create this one to open a ticket

**Reply by donerre** [reply]
after +- 24hrs trying i just got a code 3 mins before it would expire.. dont wanna give hope, but maybe u should try again ppl!

**Comment by Bahriddin-Boboyev** [comment]
I'm having this problem too :(

**Comment by mijoskutt** [comment]
Same problem here (for a long time not used account "bkcdev1")!

**Comment by ailyee** [comment]
Same

**Reply by KMcL27** [reply]
Me too

**Comment by HalfFriedPotato** [comment]
I'm having this problem too :(

**Comment by victorhugomacruz** [comment]
The same thing happens to me. The code expires in 1 hour, and only arrives in my email after 4, 5 hours... this way it is impossible to log into the account lol

**Comment by jourdyjunior** [comment]
GITHUB BOSOOOKKKK!!! FIX CEPAT LAH BIKIN STRESS DOANG

**Reply by aminulloh2002** [reply]
sabar bro wkwk

**Comment by aminulloh2002** [comment]
So there is couple hours delay from receiving the email, 
the code already expired when i received it lol

**Comment by hadysalam96** [comment]
Same issue,

**Comment by orfinex** [comment]
same problem,

**Comment by mir6ux** [comment]
same issue, been trying to login whole day, received codes hours later when they were expired already

**Comment by marcospzb** [comment]
same here.

**Comment by FrontendProOR** [comment]
resend verification message is fixed but from first try login not sure i don't want to risk it XD lol i got 30 verification messages

**Reply by victorhugomacruz** [reply]
normalized for you?

**Reply by FrontendProOR** [reply]
yeah, it got solved

**Comment by mhhany2000** [comment]
🎉this problem was fixed, I just tried and finally got back into my account 🎉 

my emails were getting sent before but they got to me too late and the code expires which I believe is the same issue.

**Comment by leoxxx** [comment]
Same. But I login no email today.

**Comment by ahmedmustafamo** [comment]
issue fixed from my end. i've received a mail and logged in

**Comment by Bahriddin-Boboyev** [comment, marked as answer]
GitHub is informed about this issue and fixed it. You can see the main information [here](https://www.githubstatus.com/incidents/b6f4j9ljz9r6).

**Comment by StubbornDeer** [comment]
Hey, the same issue - I can't log in to some account, is says the authentication code is sent, but after several tries, still don't have any (tested my email, it works normally).

**Comment by htcisd1** [comment]
Hi, i am facing the same issue now i have created this new account the problem is same, please help

**Comment by rwdr0** [comment]
My authentication codes arriving late at bulk. Can't freaking tell which one is the latest and if I try to use a code sent a few minutes before that has already been invalidated. Fix your shitty mail service @GitHub

**Comment by Hamza-324** [comment]
Having the same issue now, anyone still facing it ?

**Comment by charliedaly** [comment]
i'm having this problem today.
